### ISSUE TITLE
ldap_servers hash order

### ISSUE THREAD

**Issue by TheChuckMo** [vshn/puppet-gitlab#51]
After adding an ldap_server to my gitlab hiera config every couple puppet run will trigger a reconfig. Looking into it the puppet output is shwoing a change in the gitlab.rb file. But the diff only shows a difference in the way the ldap_server hash is written to the file. 

```
--- /etc/gitlab/gitlab.rb	2016-01-06 16:11:16.191974903 -0800
+++ /tmp/puppet-file20160106-23945-3txlqb	2016-01-06 16:20:42.223501596 -0800
@@ -21,7 +21,7 @@
 gitlab_rails['gitlab_email_display_name'] = ' Gitlab'
 gitlab_rails['gitlab_email_enabled'] = true
 gitlab_rails['ldap_enabled'] = true
-gitlab_rails['ldap_servers'] = {"myldapserver"=>{"host"=>"ldap.", "block_auto_created_users"=>false, "uid"=>"sAMAccountName", "active_directory"=>true, "label"=>"LDAP", "password"=>"", "bind_dn"=>"cn=", "base"=>"dc=", "allow_username_or_email_login"=>false, "method"=>"plain", "port"=>389}}
+gitlab_rails['ldap_servers'] = {"myldapserver"=>{"password"=>"", "block_auto_created_users"=>false, "uid"=>"sAMAccountName", "bind_dn"=>"cn=", "base"=>"dc=", "method"=>"plain", "allow_username_or_email_login"=>false, "port"=>389, "label"=>"LDAP", "host"=>"ldap.", "active_directory"=>true}}
 gitlab_rails['time_zone'] = 'america/los_angeles'
 ```

*removed content, but you can see the keys are in a different order.

**Comment by tobru** [comment]
Will be fixed with #52

**Comment by TheChuckMo** [comment]
Thank you!
